### ISSUE TITLE
can not set sharingKey with loadFlatScore

### ISSUE THREAD

**Issue by stafyniaksacha** [FlatIO/embed-client#162]
Hello,
is there a way to use embed instance to load score with a sharing key after initiation ? 

```js
const embed = new Embed("#element", {
  embedParams: {
    appId: 'xxxx',
    controlsParts: true,
  },
})

// later
embed.loadFlatScore('xxx', { sharingKey: 'xxx' })
```

**Comment by gierschv** [comment]
Thanks for your feedback. Indeed that's not possible at the moment, I will add it for the next release.

**Comment by gierschv** [comment]
Now available in 1.4.0: https://github.com/FlatIO/embed-client/tree/v1.4.0#loadflatscorescore-mixed-promisevoid-apierror
